sdk/python/tests/engine/test_cli.py

```diff
@@ -41,4 +48,8 @@ def test_cli_exec_errors(config_args: dict, call_kwargs: dict, fp: FakeProcess):
         with cli.CLISession(dagger.Config(**config_args), "dagger"):
             ...
 
-    assert "Dagger engine failed to start" in str(exc_info.value)
+    assert (
+        "Dagger engine failed to start."
+        + "If engine was manually stopped, run it again or delete it"
+        in str(exc_info.value)
+    )
```

Comment:
I don't think it's necessary to test the whole message. Just the most important bit of "Dagger engine failed to start" is enough.

Reply:
Changed solution by following Erik's advice